apps/sasquatch/src/jcenterDependency/java/com/microsoft/azure/mobile/analytics/mobile/sasquatch/features/GetLastSessionErrorReportFeatureTest.java

```diff
@@ -0,0 +1,18 @@
+package com.microsoft.azure.mobile.analytics.mobile.sasquatch.features;
+
+import android.util.Log;
+
+import com.microsoft.azure.mobile.crashes.Crashes;
+import com.microsoft.azure.mobile.crashes.model.ErrorReport;
+
+import static com.microsoft.azure.mobile.sasquatch.activities.MainActivity.LOG_TAG;
+
+public class GetLastSessionErrorReportFeatureTest {
```

Comment:
Yeah, it's a feature test, the one for get last session error report. So it's a get last session error report feature test.